docker/fusionauth/fusionauth-app/Dockerfile

```diff
@@ -63,7 +63,7 @@ RUN case "${BUILDPLATFORM}" in \
         ;;\
     esac \
     && apt-get update \
-    && apt-get install -y curl unzip \
+    && apt-get install -y --no-install-recommends ca-certificates curl unzip \
```

Comment:
Can you explain a bit more about this one. If we don't install all the recommended packages, how does that avoid a pre-build scan failure?

Reply:
Trivy dings any `apt-get install` that is missing that flag because it's an image size optimization/attack surface reduction